src/Input/TextField/TextField.tsx

```diff
@@ -10,132 +10,93 @@ import {
   TextFieldLabel,
 } from '../../Style/Input/TextFieldStyle';
 
-class TextField extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    const { type } = this.props;
-
-    this.state = {
-      floating: false,
-      inputType: type,
-    };
-  }
-
-  handleFocusChange = (
-    onBlur: (e: React.FocusEvent<HTMLInputElement>) => void
-  ) => {
-    const listener = (e: React.FocusEvent<HTMLInputElement>) => {
-      this.setState({
-        floating: e.target.value.length > 0,
-      });
-
-      if (onBlur !== undefined) {
-        return onBlur(e);
+export const isEmpty = (type: textFieldType, value: any) => {
```

Comment:
can we have a `isFilled` instead of `isEmpty`? 
So this will be easier to read `setFloating(isFilled(inputType, checkedValue));`

Reply:
this is minor.